test/e2e/testing-manifests/auth/encrypt/run-e2e.sh

```diff
@@ -137,7 +137,7 @@ main(){
     mkdir -p "${ARTIFACTS}"
 
     export GO111MODULE=on;
-    go install sigs.k8s.io/kind@v0.17.0;
+    go install sigs.k8s.io/kind@v0.21.0;
```

Comment:
Any reason to keep this pinned?

```suggestion
    go install sigs.k8s.io/kind@latest;
```

Reply:
Just so we can control the version upgrade instead of being broken by any changes with the `latest` versions that don't work with our existing config.

Reply:
But didn't we get broken because we were pinned to an old version that stopped working with the automatic bumps from https://github.com/kubernetes/test-infra/commits/master/config/jobs/kubernetes/sig-auth/sig-auth-encryption-at-rest.yaml ?

Reply:
We can get broken with both the options, I think it comes down to how often we'll get broken using `latest` vs being pinned to an old version. I'll update it to `latest` for now and we can revisit this if we have an issue with `latest.